example/satosa/pyeudiw_backend.yaml

```diff
@@ -220,14 +220,18 @@ config:
             p: 2zmGXIMCEHPphw778YjVTar1eycih6fFSJ4I4bl1iq167GqO0PjlOx6CZ1-OdBTVU7HfrYRiUK_BnGRdPDn-DQghwwkB79ZdHWL14wXnpB5y-boHz_LxvjsEqXtuQYcIkidOGaMG68XNT1nM4F9a8UKFr5hHYT5_UIQSwsxlRQ0
             q: 2jMFt2iFrdaYabdXuB4QMboVjPvbLA-IVb6_0hSG_-EueGBvgcBxdFGIZaG6kqHqlB7qMsSzdptU0vn6IgmCZnX-Hlt6c5X7JB_q91PZMLTO01pbZ2Bk58GloalCHnw_mjPh0YPviH5jGoWM5RHyl_HDDMI-UeLkzP7ImxGizrM
-    # x509:
-      # module: pyeudiw.trust.handler.x509
-      # class: X509TrustModel
-      # config:
-        # trust_anchor_certificates:
-            # - "todo"
-        # trust_anchors_cn: # we might mix CN and SAN together
-            # - http://127.0.0.1:8000
+    x509:
+      module: pyeudiw.trust.handler.x509
+      class: X509TrustHandler
+      config:
+        client_id: *client_id
+        relying_party_certificate_chains_by_ca:
+            - ca.example.com: 
+              - !!binary 0\x82\x02\xd60\x82\x01\xbe\xa0\x03\x02\x01\x02\x02\x14&\xe9\x82\xe89\xc2z\xb8\xec#\x8c\xf1~\x95d\xf3b\xb7\x97E0\r\x06\t*\x86H\x86\xf7\r\x01\x01\x0b\x05\x000\x191\x170\x15\x06\x03U\x04\x03\x0c\x0eca.example.com0\x1e\x17\r250312132316Z\x17\r260313132316Z0\x191\x170\x15\x06\x03U\x04\x03\x0c\x0eca.example.com0\x82\x01"0\r\x06\t*\x86H\x86\xf7\r\x01\x01\x01\x05\x00\x03\x82\x01\x0f\x000\x82\x01\n\x02\x82\x01\x01\x00\xd2\xd8\xcb\x8c\x8a\x86\xabF\xe4\x89\xd0HP\x0b\x12.\x87\xb2\xbfL\x06)\x15\x12\xc9+D\xc8\xe5\xdc\x8fu\xd9\xac\xccdke\xa6\x87\xb7\x80\x1ek\xb8\xc6\xa2\x1aL>\xf5Q\xd7\x05\xc32g\xc6\n\xb6\xf5\xc9)\x1e\x19L\xf0\xac\xc6\xda\xf4\xb3\x11L\xac%|\xc1\xf1\xbaMP/\xd9F\x94k\xf0l\xa7\r\n\xd1a)Y\x08\xf8\xff\xe2\x907\xa4\xa3\xef\xbb\xf2\x07\x1a\xfeo\xedn\x9f\xb9\xc0\x9eh\xf7\xfc\xa2G|\xa0Z{l\xa3\x844y$\xf3\x1c\xe0\xac\xa5\xa9GG}\xbb\xac\x15\xaa\xcc\xde\xbcr\xcb\xc7\x19\x80\xb8\xfe\xbc\x06I[\xfa\x9f\xdbc\xe5\x91\xd6^\xabP\x9d<\xf1\x8dH\x7f\xef\x11S\xf7\xe0\xd1\xbc\xc5R\xe0\xb9\xdb\xae\\__w\xe3\x93\xb9\x1f\x942V\x9e\xd6\xec\xacM\n-#\xa9\nV,\xd0\xb06\x05T\xf6\xb3K<G\x00\x07\xcb\xc3\xe7\x97K\x11\xf02\xbcQ\xd3\xbe:D\x1aWg\x8b\x8a\x8d,\xd0\xfa\xfc\xf9\xec\t.\x8e4\xe95\xa3\x02\x03\x01\x00\x01\xa3\x160\x140\x12\x06\x03U\x1d\x13\x01\x01\xff\x04\x080\x06\x01\x01\xff\x02\x01\x010\r\x06\t*\x86H\x86\xf7\r\x01\x01\x0b\x05\x00\x03\x82\x01\x01\x00~\xb4.{h\x88\x1f\xc7KV#JIYy\x8c\xf32\x8b\xb4\xdak\xde\xbe\x03\x9cZ\xea9\xb6P\xc8\xfd.\x18\xf4\x1e\xcfv\xb8\xfc\xc2]\x9a\x1f\x01\x13g\xbf+\xbe\xeb\x16\x11=nS\xdf\xb6\x1b`87\x9c\x1c\'b\xee}~\xe6\r\x86\xec\x8e\xc1\x03\xb4e\x0fN\xbf\n\xe0\xca!\xc4O~\x93\xa9\t\x12.\xd3\x0b0\xb9\xe8\xd9b\xe1\x10\x0eS^d!\xd9\xf7}\x15c;\xa2J\x82\xac\x83\xa9c/\xc2\xa9\xcb\xdb\xda\x06\x1a\xb6J\xb8 vu\x82\x9b\x1f>\x8e]\xdd\x05\xe5E@F+Z\xa0&&\x1f\xc7Y\xd3*J\'\x8d\xcc\xa7\xb7g\xcd\x10\x96#R\t\x92\xc6\xb44F\xdf\xb3\xb7\xd9&<\xab\xdaC~\xb8@\x98\x91\xf2\x17>,\xdd\xfe\xe6\xa8%\xd5E\xa7d\x80\xac\t\xe4U\x02\xa2i\xd9\x13\xfb\xc6e\x19{\xabnZ8A\xc6\xd9mN\xec\x99\xb6))1y\xca\x94\x02\x9f\xc5\xc2g\x87\xdf\x91\x85\x08O\xb3\xcb,\xed\xb2j!\n\R
```

Comment:
please enable PEM format as string too

this is the most popular serialization format for X.509 certificates

Reply:
handled in the code